### DISCUSSION TITLE
Official docker container

### DISCUSSION THREAD

**Discussion by johnpangalos** [esm-dev/esm.sh#663] [Ideas] [answered]
Hey, I was wondering if esm.sh would be interested in making an official docker image of it's go code for self hosting. This would be useful if you're trying to use esm.sh for private repositories.

[I've made one myself for now.](https://github.com/johnpangalos/docker-esm-dot-sh) Please feel free to use it or if esm.sh is interested take it over I would be open to that as well. Thanks!

**Comment by ije** [comment]
looks great😍  I will try to setup with your files 🙏

**Reply by johnpangalos** [reply]
Let me know if you need any help with it, the updating of the docker containers are triggered by a cron job and preferably they'd be triggered by [tag generation or push to the main branch](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore).

**Reply by ije** [reply]
thanks 👍

**Comment by ije** [comment, marked as answer]
hi @johnpangalos i added an official docker image based on your version, thank you so much!

```bash
docker pull ghcr.io/esm-dev/esm.sh
```
